test/unit/spec/common/cellComponents/fieldTableCellWidget-Spec.js

```diff
@@ -5,21 +5,21 @@ define([
     'common/props',
     'common/spec',
     '/test/data/testAppObj',
-], (Jupyter, fieldCellWidget, ParamResolver, Props, Spec, TestAppObject) => {
+], (Jupyter, FieldCellWidget, ParamResolver, Props, Spec, TestAppObject) => {
     'use strict';
 
     describe('The Field Table Cell Widget module', () => {
         it('loads', () => {
-            expect(fieldCellWidget).not.toBe(null);
+            expect(FieldCellWidget).not.toBe(null);
         });
 
         it('has expected functions', () => {
-            expect(fieldCellWidget.make).toBeDefined();
+            expect(FieldCellWidget.make).toBeDefined();
         });
     });
 
     describe('The Field Table Cell Widget instance', () => {
-        let node, mockFieldWidget;
+        let node, fieldCellWidgetInstance;
```

Comment:
standardisation: name the objects produced by a module `${moduleName.toCamelCase()}.Instance`